01-arch/02-policy.md

```diff
@@ -1 +1,116 @@
 ## Storage Policy
+
+In NeoFS, storage policy is a flexible way to specify rules for storing objects.
+The result of storage policy rules application to the network maps is a set of
+storage nodes, able to store the data according to the requested policy. This
+result maybe called "placement", hence sometime you may find the term "placement
+policy" used to denote the same thing as the "storage policy".
+
+Because Storage Policy is attached to the container structure there is a compact
+definition for system's internal use and some higher level language definitions
+for humans to use, that are translated to internal representation. For example
+there is SQL-like language to use by humans, JSON notation to be used in
+software and there may be more, for example a graphical language using
+[Blockly](https://developers.google.com/blockly). In our examples we will use
+SQL-like notation.
+
+Storage Policy internal definition consists of four parts:
+
+1. Filters
+2. Selectors
+3. Replicas
+4. Container Backup Factor
+
+The result of applying a storage policy to the netmap is a set of nodes
+structured by `Replicas` and is used to select candidates to put objects on. The
+selection algorithm is deterministic, hence on different nodes or clients the
+same Storage Policy applied to the same version on netmap will give the
+identical result.
+
+### Filters
+
+Filter is a mechanism to specify which nodes are allowed to store an object.
+This is done by querying node's attributes and checking if they satisfy certain
+condition. For example, it allows to precisely specify "Store objects in nodes
+from Europe, but not from Italy, which have SSD and have a good reputation". 
+
+Simple filter can compare a single node attribute with some value.
+
+It has 3 fields:
+
+1. `Key` — name of node attribute
+2. `Value` — value to compare attribute with
+3. `Op` — operation to be used for comparison
+
+For better understanding, simple filters will be specified as `Key Op Value`.
+
+For example:
+
+1. `Country = Argentina` means use nodes for which `Country` attribute
+   is equal to `Argentina`, i.e. nodes located in USA.
+2. `Rating > 4.5` means use nodes with rating better then 4,5
```

Comment:
4.5 - in English we tend to use full stop as decimal separator (at least, use the same everywhere through the text)